test/evergreen.yml

```diff
@@ -2979,9 +2979,9 @@ tasks:
             # Set the cyclomatic complexity limit to 20
             python "../metrixplusplus/metrix++.py" limit --max-limit=std.code.complexity:cyclomatic:20
 
-            # Fail if there are functions with cyclomatic complexity larger than 95
+            # Fail if there are functions with cyclomatic complexity larger than 91
```

Comment:
Can we add this task to the list of PR tests? That way we can catch future violations before they're merged into dev.

Reply:
Yes, we should